Libplanet.Tests/EnumerableShim.cs

```diff
@@ -0,0 +1,26 @@
+#pragma warning disable S1128
```

Comment:
Yeah, this is warned only on .NET Core, because the `EnumerableShim.Append()` definition is commented (due to the directive `#if NETFRAMEWORK && !NET48 && !NET472 && !NET471`) out on it.  On Mono, this is not warned as the definition is visible to the compiler.